source/install/mattermost-omnibus.rst

```diff
@@ -0,0 +1,117 @@
+Installing Mattermost Omnibus
+============================
+
+Overview
+---------
+
+Mattermost Omnibus is a Debian package that packages the individual components of a Mattermost deployment into a single installation process. The package leverages the ``apt`` package manager to install and keep the components of the platform updated, using a custom CLI and ansible recipes to link those components together and configure them.
+
+Mattermost Omnibus currently supports Ubuntu's ``bionic`` and ``focal`` distributions. Support for RedHat/CentOS distributions will be available in future versions. 
+
+Prerequisites
+-------------
+
+- A clean Ubuntu server (18.04. Or 20.04)
+- Root level access to the server
+
+.. note:: 
+
+  With Mattermost Omnibus, the ``config.json`` file no longer exists. So you’ll need to use mmctl to make changes to your Mattermost server configuration using ``mmctl config edit``. If you wish to manage your server locally, you may want to enable local mode for mmctl.
+
+Installing Mattermost Omnibus
+------------------------------
+
+During installation you can provide an existing domain name that points to the Ubuntu server or, if you don't have one available, you can log in using the server's IP address.
+
+.. note::
+  Plugin uploads and HTTPS are enabled by default. These settings are modified in the ``yaml`` file using `mmctl config edit <https://docs.mattermost.com/administration/mmctl-cli-tool.html#mmctl-config-edit>`__. 
+
+The Omnibus repositories are configured using a cURL command: 
+
+``curl -o- deb.packages.mattermost.com/repo-setup.sh | sudo bash``
+
+This command installs and configures a ``PostgreSQL`` database, an ``nginx`` web server to act as a proxy, and ``certbot`` to issue and renew the SSL certificate. 
+
+In order to issue that certificate, the installer requests a domain name and an email address from us. These are used to generate the certificate and deliver any related communications respectively.
+
+Installing Omnibus Package and Mattermost Server
+------------------------------------------------
+
+If you're installing Omnibus, and have a domain and SSL use: 
+
+``sudo apt install mattermost-omnibus``.
+
+After all the packages are installed, Omnibus runs the ansible scripts that configure all the platform components and starts the server. To access Mattermost, open a browser, navigate to your domain, and create the System Admin user to start using the platform. 
+
+You can also install Mattermost without a domain name or SSL (not recommended). To do this use: 
+
+``sudo MMO_HTTPS=false apt install mattermost-omnibus``
+
+Updating Mattermost Omnibus
+-----------------------------
+
+Mattermost Omnibus is integrated with Ubuntu’s OS's package manager. When a new Mattermost version is released, run:
```

Comment:
This is correct, but just in case we decide to start supporting other APT based distros, we can reference APT directly, like:

```suggestion
Mattermost Omnibus is integrated with the ``apt`` package manager. When a new Mattermost version is released, run:
```